sbi/inference/posteriors/likelihood_based_posterior.py

```diff
@@ -461,14 +485,15 @@ def posterior_potential(
         theta = ensure_theta_batched(torch.as_tensor(theta, dtype=torch.float32)).to(
             self.device
         )
+        theta, log_abs_det = self.transform(theta)
 
         log_likelihoods = LikelihoodBasedPosterior._log_likelihoods_over_trials(
```

Comment:
like in `direct posterior`, the likelihoods we get here are in untransformed space, no? I would make that explicit and then add log abs det explicitly to move them to transformed space.